sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/SpannerWriteGroupFn.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Would you prefer if I send https://github.com/apache/beam/pull/3395/commits/e9528e16a4e3a9a8b8b352fcd2183c505f32bfa9 as a separate RP?